Vagrantfile

```diff
@@ -8,15 +8,17 @@ require "#{VAGRANTFILE_DIR}/vagrant/lib/forklift"
 
 def migrate_boxes!
   old = "#{VAGRANTFILE_DIR}/boxes.d/99-local.yaml"
+  older = "#{VAGRANTFILE_DIR}/boxes.yaml"
   new = "#{VAGRANTFILE_DIR}/vagrant/boxes.d/99-local.yaml"
 
-  return if File.symlink?(old) || !File.exist?(old)
+  return if File.symlink?(old) || !File.exist?(old) || !File.exist?(older)
 
   if File.exist?(new)
     raise "File #{new} already exists, refusing to overwrite. Remove boxes.d/99-local.yaml in favor of vagrant/boxes.d/99-local.yaml"
   end
 
-  File.rename(old, new)
+  File.rename(older, new) if File.exist?(older)
```

Comment:
I'd still add `&& !File.symlink?(older)` and below.

Reply:
Wouldnt that mean `old` doesn't trigger thus allowing older to win? And I moved the `old` after so that `old` wins?

Reply:
Perhaps it's easier to write it in a loop or in 2 steps? Less mental accounting.

Reply:
This is starting to feel overly complex to be honest. I have to now account for symlinks someone may have created that are not part of our documentation. Given this is the oldest possible format and its been many months we've changed to the new I'm leaning away from doing this at all.

Reply:
I'm fine with that tbh. We have no formal release or whatever.